tests/date.test.js

```diff
@@ -0,0 +1,27 @@
+import { formatDate } from '../utils/date'
+import dayjs from 'dayjs'
+
+describe('formatDate function', () => {
+    it('sets the short date format as a default value', () => {
+        expect(formatDate('2019-09-25 13:15:40.000000')).toMatch('25.09.2019')
+    })
+
+    it('sets the long date format if the "long" parameter is provided', () => {
+        expect(formatDate('2019-09-25 13:15:40.000000', 'long')).toMatch('25.09.2019, 13:15 Uhr')
+    })
+
+    it('transforms number to a string using \'DD.MM.YYYY\' format', () => {
+        expect(formatDate(1569369600000)).toMatch('25.09.2019')
+    })
+
+    it('transforms number to a string using a long \'DD.MM.YYYY, HH:mm [Uhr]\' format ', () => {
+        expect(formatDate(1569410140000, 'long')).toMatch('25.09.2019, 13:15 Uhr')
+    })
+
+    it('sets the current date if the provided date is a null or undefined', () => {
```

Comment:
```suggestion
    it('sets the current date if the provided date is a string or instanceof date', () => {
```
you can check this, too. it would be nice. and the cases what happens if you give other type of dates for example german date or short german or english date

you forgot here to check the  toDate function. there you can check what happens if there is any other date format

Reply:
Thanks for the advice. I've refactored tests for the formatDate function. done here https://github.com/demos-europe/demosplan-js-utils/pull/10/commits/f9b288a3399b7a842b3630ec3bbe92a042ad53a3

Reply:
also added some tests for `toDate` function here https://github.com/demos-europe/demosplan-js-utils/pull/10/commits/303c7a805ff80ac1c584728946bd1b3f08c65fa5